<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogShedule</class>
 <widget class="QDialog" name="DialogShedule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>427</width>
    <height>495</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>427</width>
    <height>495</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>427</width>
    <height>495</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Shedule</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>420</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(184, 216, 239)</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="doc.qrc">:/doc/990282.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 57 italic 40pt &quot;URW Chancery L&quot;;</string>
   </property>
   <property name="text">
    <string>Shedule</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>161</width>
     <height>276</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="mondaycheckBox">
      <property name="styleSheet">
       <string notr="true">font: 57 italic 16pt &quot;URW Chancery L&quot;;
</string>
      </property>
      <property name="text">
       <string>Monday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="tuesdaycheckBox">
      <property name="styleSheet">
       <string notr="true">font: 57 italic 16pt &quot;URW Chancery L&quot;;</string>
      </property>
      <property name="text">
       <string>Tuesday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="thursdaycheckBox">
      <property name="styleSheet">
       <string notr="true">font: 57 italic 16pt &quot;URW Chancery L&quot;;</string>
      </property>
      <property name="text">
       <string>Thursday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="wednesdaycheckBox">
      <property name="styleSheet">
       <string notr="true">font: 57 italic 16pt &quot;URW Chancery L&quot;;</string>
      </property>
      <property name="text">
       <string>Wednesday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="fridaycheckBox">
      <property name="styleSheet">
       <string notr="true">font: 57 italic 16pt &quot;URW Chancery L&quot;;</string>
      </property>
      <property name="text">
       <string>Friday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="saturdaycheckBox">
      <property name="styleSheet">
       <string notr="true">font: 57 italic 16pt &quot;URW Chancery L&quot;;</string>
      </property>
      <property name="text">
       <string>Saturday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="sundaycheckBox">
      <property name="styleSheet">
       <string notr="true">font: 57 italic 16pt &quot;URW Chancery L&quot;;</string>
      </property>
      <property name="text">
       <string>Sunday</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="doc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogShedule</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogShedule</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
